### PR TITLE
Read IGRF coefficients from file

### PR DESCRIPTION
**Pull request by sputnik-a**
IGRF coefficients are directly read from ASCII-file instead of using
hard-coded values. This omits adding the coefficients manually in
the future.

### CODE REVIEW

**Comment by asreimer** [comment]
Hi @sputnik-a,

Can you please redo this PR? Start from the develop branch (as per [contributing guidelines](https://github.com/aburrell/apexpy/blob/master/CONTRIBUTING.rst)? You should be able to simply cherrypick the commits you made here.

Thanks!

File: AUTHORS.rst [hunk 1]
```diff
@@ -7,6 +7,8 @@ This python wrapper is made by:
 * Karl M. Laundal
 * Christer van der Meeren
 * Angeline G. Burrell (maintainer)
+* Ashton Reimer
+* Gregory Starr
```
Comment (aburrell) [inline AUTHORS.rst:11]: Please add your name to the end of the Author list.

**Review by aburrell — CHANGES_REQUESTED** [review]
I rebased the pull request and resolved the conflicts.  I verified the coefficient file, but am having trouble getting the fortran code to work (and thereby having trouble getting the python to install).  I am going to try and rope Emmert in on this, but do feel free to help with the troubleshooting.

**Review by aburrell — APPROVED** [review]
This looks great, thank you!

**Comment by sputnik-a** [comment]
> This looks great, thank you!

Thanks! I am glad everything is working now. Sorry I was busy the last weeks - let me know when I can do anything else with the Fortran part in the future or in case of problems with the IGRF part.

**Pull request merged**